src/ansible_navigator/runner/__init__.py

```diff
@@ -0,0 +1,8 @@
+""" Defined public entrypoints for the runner api
+"""
```

Comment:
Modules are static, their docstrings should not define actions but describe what the thing is. In this case, it's the whole package docstring and it should describe what is `ansible_navigator.runner`.
```suggestion
"""Public entry points for the runner API."""
```

Reply:
Fixed in #630